.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
 
       - name: Dry run release
         env:
-          GITHUB_TOKEN: ${{ steps.app.outputs.token }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You changed the token here, is that intended?

Reply:
Yepz, dry run should never read, so use the bot token here which can't write in protected branches.

Reply:
See also https://github.com/philips-software/app-token-action/pull/4/checks?check_run_id=2080506136